orangecontrib/bio/widgets3/OWGenExpress.py

```diff
@@ -776,9 +781,17 @@ def Commit(self):
             if self.joinreplicates:
                 table = dicty.join_replicates(table,
                     ignorenames=self.dbc.IGNORE_REPLICATE,
-                    namefn=None,
+                    namefn=lambda att: '{}HR'.format(att['Timepoint']),
```

Comment:
Why this? This only leaves in the timepoint information. What if we are joining replicates for different mutants at the same timepoint?